### PR TITLE
Added support for unconditional_checked_cast_addr instruction

### PR DESCRIPTION
**Pull request by mmroz**

### CODE REVIEW

File: lib/WALASupport/SILWalaInstructionVisitor.cpp [hunk 1]
```diff
@@ -1939,6 +1939,25 @@ jobject SILWalaInstructionVisitor::visitUncheckedOwnershipConversionInst(Uncheck
...
 /*******************************************************************************/
 
+jobject SILWalaInstructionVisitor::visitUnconditionalCheckedCastAddrInst(UnconditionalCheckedCastAddrInst *CI) {
+  SILValue SrcValue = CI->getSrc();
+  SILValue DestValue = CI->getDest();
+
+  if (Print) {
+    llvm::outs() << "\t [UnconditionalCheckedCastAddrInst]: " << CI << "\n";
+    llvm::outs() << "\t [CONVERT]: " << CI->getSourceType().getString() << " " << SrcValue.getOpaqueValue();
+    llvm::outs() << " [TO]: " << CI->getTargetType().getString() << " " << DestValue.getOpaqueValue() << "\n";
```
Comment (karimhamdanali) [inline lib/WALASupport/SILWalaInstructionVisitor.cpp:1949]: should there be a "\t" at the beginning of this info line too?

Comment (mmroz) [reply lib/WALASupport/SILWalaInstructionVisitor.cpp:1949]: No, its meant to print on a single line it is just very long when put on a single line

Comment (karimhamdanali) [reply lib/WALASupport/SILWalaInstructionVisitor.cpp:1949]: Ah I see. Approving changes.

**Review by karimhamdanali — APPROVED** [review]
LGTM

**Pull request merged**